README.md

```diff
@@ -28,10 +28,28 @@ After you upgrade, you'll need to merge back any customizations you made to `App
 
 *Otherwise*
 
-* Download the latest version of [Application Insights Java SDK](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started), [scroll down to section](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started)and follow the instructions to manually download the SDK and replace the old Jars.
+* Download the latest version of [Application Insights Java SDK](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started), [scroll down to the getting started section](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started) and follow the instructions to manually download the SDK and replace the old `.jar` files.
  
 Compare the old and new `ApplicationInsights.xml`. Many of the changes you see are because we added and removed modules. Reinstate any customizations that you made.
 
+## Application Insights for Java Roadmap
+The Application Insights team have been hard at work to deliver the next wave of features for our Java support and experience. Below is an outline of the features and improvements that are planned for our next release, which is targeted to be completed for mid-year 2018.
+
+### Application Insights for Java SDK 2.0
+In December [we released the beta version of the Application Insights for Java 2.0 SDK](https://github.com/Microsoft/ApplicationInsights-Java/releases/tag/v2.0.0-BETA). That release introduced support for cross-component telemetry correlation and fixed rate sampling. For the final release, we'll be addressing any reliability issues and improving performance.
```

Comment:
We're not "improving" performance with stable release. Rather we've conducted perf tests in order to determine the perf overhead of the SDK. As results were satisfactory, we did not make any perf-related code changes.

Reply:
Yes, this is true. I thought it was implied. If needed we can revert this merge or just file another PR.

Reply:
Cool, sounds good.